### DISCUSSION TITLE
Formatted value ignoring negligible decimal places

### DISCUSSION THREAD

**Discussion by TooColline** [dm4t2/vue-currency-input#312] [Q&A]
## Vue Currency Input version
2.4.0

## Vue version
2.6.14

I need the formatted value to not ignore negligible decimal places. If I want to input `3,50` for instance I need it to be as it is, not sure if this is a possible feature request. My chosen precision is: `precision: { min: 0, max: 5 },`

**Comment by dm4t2** [comment]
Maybe I'm missing something, but isn't `precision: { min: 2, max: 5 }` a solution for your example?

**Reply by TooColline** [reply]
It could be but there are instances where if I need the user to input `3,540` or `3,500` or just `3,` but still maintain a maximum of 5 decimal places. Having `min: 0` doesn't allow this for some reason

**Reply by TooColline** [reply]
@dm4t2 I would appreciate your response on the above when you get the chance

**Reply by dm4t2** [reply]
Your requirement is still unclear. Please provide more accurate examples to describe the actual and expected behavior. Thanks!
